pgModel/main.go

```diff
@@ -0,0 +1,53 @@
+package pgModel
+
+import (
+	"fmt"
+	"github.com/go-pg/pg"
+	"github.com/subosito/gotenv"
+	"os"
+	"golang.org/x/crypto/bcrypt"
+)
+
+func HashPassword(password string) (string, error) {
+	bytes, err := bcrypt.GenerateFromPassword([]byte(password), 14)
+	return string(bytes), err
+}
+
+func CheckPasswordHash(password, hash string) bool {
+	err := bcrypt.CompareHashAndPassword([]byte(hash), []byte(password))
+	return err == nil
+}
+
+ 
+ func (u User) String() string {
+	return fmt.Sprintf("User<%d %s %v>", u.Id, u.Username, u.Email)
+ }
+
+ func Connect() *pg.DB {
+	gotenv.Load()
+
+	db := pg.Connect(&pg.Options{
+		User: "postgres",
+		Password: "dafe",
```

Comment:
@mikey2020 what do you think about us adding User and Password to .env file?
- They are variables as much as the Database
- And I feel they are meant to be confidential credentials. We don't want unsolicited access to our database.

Reply:
forget about those will changes it